dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -962,6 +962,26 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
     assert_nil signed_in_user_id
   end
 
+  test 'login: google_oauth2 does not trigger silent take over on migrated Clever student with multiple credentials' do
+    email = 'test@foo.xyz'
+    uid = '654321'
+    user = create(:student, :migrated_imported_from_clever, uid: uid)
+    user.authentication_options << create(:google_authentication_option, user: user, email: email, authentication_id: uid)
+    user.save
+    auth = generate_auth_user_hash(provider: 'google_oauth2', uid: uid, user_type: User::TYPE_STUDENT, email: email)
+    @request.env['omniauth.auth'] = auth
+    @request.env['omniauth.params'] = {}
+
+    assert_does_not_destroy(User) do
+      get :google_oauth2
+    end
+    user.reload
+    assert_equal 'migrated', user.provider
+    auth_option_count = user.authentication_options.count
+    assert_equal 2, auth_option_count
```

Comment:
Could you add a comment explaining what 2 authentication options we are expecting? Do we expect `get :google_oauth2` not creating a new authentication option?